package.json

```diff
@@ -76,5 +76,68 @@
     "dev-packages/*",
     "packages/*",
     "examples/*"
-  ]
+  ],
+  "pluginsDir": "plugins",
```

Comment:
could we omit pluginsDir and default to `plugins` if not specified ?

Reply:
> could we omit pluginsDir and default to `plugins` if not specified ?

The default is `plugins` if not specified :)